packages/matter.js/src/behavior/definitions/operational-credentials/OperationalCredentialsServer.ts

```diff
@@ -423,6 +430,12 @@ export class OperationalCredentialsServer extends OperationalCredentialsBehavior
         await this.context.transaction.commit();
     }
 
+    async #assureCertification() {
```

Comment:
Given that every call to #certification is prefaced by #assureCertification seems simpler if you combine #assureCertification and #certification into one method. That'd also make it impossible to introduce logic errors in the future by omitting #assureCertification

Reply:
#Sometimes there is multiple accesses to #certification after the assure call and I would then need to make these all Async. I decided for that way because so it is one please which ensure it using await so the other access can be sync.

Reply:
Thx. I will sleep over it ;-)